coordinator/src/metrics.rs

```diff
@@ -64,12 +76,88 @@ pub fn init_meter() -> PrometheusExporter {
 
 pub fn collect_metrics(node: Node) {
     let cx = opentelemetry::Context::current();
+    position_metrics(&cx, &node);
+
     let inner_node = node.inner;
     let channels = inner_node.channel_manager.list_channels();
     channel_metrics(&cx, channels);
     node_metrics(&cx, inner_node);
 }
 
+fn position_metrics(cx: &Context, node: &Node) {
+    let mut conn = match node.pool.get() {
+        Ok(conn) => conn,
+        Err(e) => {
+            tracing::error!("Failed to get pool connection. Error: {e:?}");
+            return;
+        }
+    };
+
+    let positions = match db::positions::Position::get_all_open_positions(&mut conn) {
+        Ok(positions) => positions,
+        Err(e) => {
+            tracing::error!("Failed to get positions. Error: {e:?}");
+            return;
+        }
+    };
+
+    let mut margin_long = 0;
+    let mut margin_short = 0;
+    let mut quantity_long = 0.0;
+    let mut quantity_short = 0.0;
+
+    // Note: we should filter positions here by BTCUSD once we have multiple contract symbols
```

Comment:
the easiest way to not forget is to put the following inside the loop instead of this comment:
 
```rust
debug_assert!(position.contract_symbol == ContractSymbol::BTCUSD' "we should filter positions here by BTCUSD once we have multiple contract symbols`)
```

It will be **not** present in production, but will blow up right after someone enables different contract symbols on their dev machine 🤭